elastica/boundary_conditions.py

```diff
@@ -258,28 +260,31 @@ class OneEndFixedRod(OneEndFixedBC):
     )
 
 
-class FixedConstraint(ConstraintBase):
+class ConfigurableConstraint(ConstraintBase):
```

Comment:
```suggestion
class GeneralConstraint(ConstraintBase):
```

Reply:
Implemented in https://github.com/GazzolaLab/PyElastica/pull/143/commits/4c06b0e850b5d5fb647394771572c35a9f0c641e